spring-cloud-skipper-shell-commands/src/main/java/org/springframework/cloud/skipper/shell/command/support/YmlUtils.java

```diff
@@ -36,29 +37,9 @@
  */
 public abstract class YmlUtils {
 
-	private static final String APPLICATION_PROPERTIES_PREFIX = "spec.applicationProperties.";
-
-	private static final String DEPLOYMENT_PROPERTIES_PREFIX = "spec.deploymentProperties.";
-
-	private static final String SPEC_APPLICATION_PROPERTIES_REPLACEMENT = "REPLACE_APPLICATION_PROPERTIES";
-
-	private static final String SPEC_DEPLOYMENT_PROPERTIES_REPLACEMENT = "REPLACE_DEPLOYMENT_PROPERTIES";
-
-	private static final String DOT_CHAR = "\\.";
-
-	private static final String DOT_CHAR_REPLACEMENT = "------";
-
-	private static final String DOT_SPEC_STRING = ".spec.";
-
-	private static final String SPEC_STRING = "spec.";
-
 	public static String convertFromCsvToYaml(String propertiesAsString) {
```

Comment:
We can probably get rid of this method as this util class is for handling just with the use-case of parsing the properties on install/upgrade with the 'flattening' so it can be easily passed into a spring cloud deployer implementation.

Reply:
Yeah, I was trying to keep minimal changes.